src/Mod/Material/Gui/BaseDelegate.cpp

```diff
@@ -354,10 +354,11 @@ void BaseDelegate::setModelData(QWidget* editor,
         value = chooser->fileName();
     }
     else if (type == Materials::MaterialValue::Quantity) {
-        auto input = dynamic_cast<Gui::InputField*>(editor);
+        auto input = dynamic_cast<Gui::QuantitySpinBox*>(editor);
```

Comment:
It's not important to fix here, but some static analyzers will note the use of dynamic_cast without checking for a null result. You could combine the variable definition into the if clause to do that check without actually adding any lines of code.

Reply:
Good point. I'll be back in there for other things so I'll fix it then.